cli/medperf/commands/compatibility_test.py

```diff
@@ -220,3 +225,20 @@ def download_demo_data(self):
         data_path = os.path.join(untar_path, paths["data_path"])
         labels_path = os.path.join(untar_path, paths["labels_path"])
         return data_path, labels_path
+
+    def cached_result(self):
```

Comment:
A docstring here explaining the intention of the function, and in which cases it returns None would be useful here